### PR TITLE
TMA-904: allow to set VCR record mode from environment

### PR DESCRIPTION
**Pull request by jimirocks**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/41/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/854/ : SUCCESS in 2m 15s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1122/ : SUCCESS in 33s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/42/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/855/ : SUCCESS in 37s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1123/ : SUCCESS in 2m 16s

File: spec/vcr_configurer.rb [hunk 1]
```diff
@@ -32,7 +32,9 @@ def self.filter_header(vcr_config, part, header_name)
         vcr_config.configure_rspec_metadata!
         vcr_config.default_cassette_options = {
           :decode_compressed_response => true,
-          :match_requests_on => [:path, :method, :query]
+          :match_requests_on => [:path, :method, :query],
+          # allow to set record mode from environment, see https://relishapp.com/vcr/vcr/v/3-0-3/docs/record-modes
+          :record => (ENV['VCR_RECORD_MODE'] && ENV['VCR_RECORD_MODE'].to_sym) || :once
```
Comment (panjan) [inline spec/vcr_configurer.rb]: This allows us to set the record mode. However, we cannot fully disable recording. Would it be useful to add this option?

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/17/ : SUCCESS in 19s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/865/ : SUCCESS in 30s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1133/ : SUCCESS in 2m 05s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1348/ : SUCCESS in 56m 04s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1111/ : SUCCESS in 34m 26s

**Pull request merged**